src/FBAConsensus.py

```diff
@@ -24,8 +24,7 @@ def get_events(cls):
         events = [Event('mine'),
                   Event('retrieve_transaction_from_mempool'),
                   Event('nominate'),
-                  Event('retrieve_message_from_peer'),
-                  Event('prepare')]
+                  Event('retrieve_message_from_peer')]
 
         # # TODO: Remove gossip event from the consensus!
```

Comment:
We have to extract it to the separate ENUM for less error-prone codebase. I will assign a separate task for these purposes